src/main/clojure/clara/benchmark/accum_repeatedly_not_selected.clj

```diff
@@ -0,0 +1,16 @@
+(ns clara.benchmark.accum-repeatedly-not-selected
+  "Added to test performance improvements in clara-rules issue 182"
+  (:require [clara.rules :refer :all]
+            [clara.rules.accumulators :as acc])
+  (:import [clara.benchmark.beans Discount Customer]))
+
+(def max-id (acc/min (fn [^Customer c] (.getId c)) :returns-fact true))
+
+(defrule vip-discount
+  [?customer <- max-id :from [Customer]]
+  =>
+  (apply insert! (repeat 1000 (Discount. 10))))
```

Comment:
This is being timed by criterium and it can skew your numbers.  Avoid doing lazy things and use the fastest possible to compete with the for-loop in the Drools case.

I'd recommend something like:
```clj
(->> (range 1000)
     (reduce (fn [coll _]
               (conj! coll (Discount. 10)))
             (transient []))
     persisent!
     insert-all!)
```